vnext/Microsoft.ReactNative/ReactHost/CrashManager.cpp

```diff
@@ -0,0 +1,99 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#include "CrashManager.h"
+#include "ReactInstanceWin.h"
+
+#include <winrt/Windows.Storage.h>
+
+#include <WerApi.h>
+
+namespace Mso::React {
+
+// when calling SetUnhandledExceptionFilter the previous filter is returned
```

Comment:
nit: The comment is a little bit confusing to read. Does it miss upper case letters to start sentenses and dots to finish them?